docs/walkthroughs/getting-started-installation.adoc

```diff
@@ -0,0 +1,62 @@
+[[getting-started]]
+= Getting started: Installation
+
+[[overview]]
+= Overview
+
+The aim of this document is to get you to the point where you have installed and enabled the mobile pieces and setup required configuration that will make mobile development against OpenShift as simple and accessible as possible. It will also explain how you can verify your OpenShift installation is ready after to start developing your mobile application against once the installation has been completed.
```

Comment:
Just wording on this sentence. Maybe something like `It will also explain how you can verify if your Openshift installation is ready so you can start developing your mobile application, once the installation has been completed.`?